styles/custom.css

```diff
@@ -13,6 +13,9 @@
   .app-nav {
     margin-right: 60px !important;
   }
+  #docsify-darklight-theme{
+    top: 6px !important;
```

Comment:
There is not much CSS in this project, why the use of `!important`

Are we able to accomplish the fix without it?

Reply:
i have tried and it does not work 
in the css file there is not much CSS but once you inspect the website u'll see all the CSS
so that is why i use `important`
**thank you for your reply**